frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -95,6 +102,28 @@ export const validateOperatingSystem: VmSettingsValidator = (field) => {
   return asValidationObject(`Select matching for: ${guestFullName}`, ValidationErrorType.Info);
 };
 
+const memoryValidation: VmSettingsValidator = (field, options) => {
```

Comment:
can we make a more general method ( e.g `numericValidation` ? ) and make this "memoryValidation" call the more general methd ?

Reply:
I'm afraid not. most of the code in this method is memory specific.

Reply:
I was thinking of a method like that (not necessarily only for numbers):
``` js
const runVlidations = (validations, value: any) => (
     validations.once( switch (validationType) {
          case min: // do one min validation at a time )
          case max: // ...
          case regexp: // ...
          // ...
    }
) 
```
and calling it in line 108, somthing like:
``` js
const valid = (validations === [] || !memValueGB) && runVlidations(vaidations, memValueGB)
```
does something like that make sense to you ?